src/index.html

```diff
@@ -11,7 +11,201 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class = "form-field">
+          <label>
+            Surname:
+            <input
+             type="text"
+             name="surname"
+             autocomplete="off"
+             required
+            >
```

Comment:
```suggestion
            <input
              type="text"
              name="surname"
              autocomplete="off"
              required
            >
```
There should be 2 spaces indentation.